cmd/fleet/serve_test.go

```diff
@@ -245,55 +243,42 @@ func TestCronVulnerabilitiesCreatesDatabasesPath(t *testing.T) {
 	}, 5*time.Minute, 30*time.Second)
 }
 
-func TestCronVulnerabilitiesMkdirFailsIfVulnPathIsFile(t *testing.T) {
-	buf := new(bytes.Buffer)
-	logger := kitlog.NewJSONLogger(buf)
-	logger = level.NewFilter(logger, level.AllowDebug())
+func TestScanVulnerabilitiesMkdirFailsIfVulnPathIsFile(t *testing.T) {
+	t.Run("mkdir fails if VulnPath is file", func(t *testing.T) {
```

Comment:
Why is it a subtest?

Reply:
Ah was planning on naming this `TestScanVulnerabilities` and using the subtest to provided a little bit more context but forgot to rename it :P - but I guess this test is simple enough to require that, I'll revert the change.